README.md

```diff
@@ -342,6 +342,8 @@ No modules.
 | <a name="input_default_route_table_propagating_vgws"></a> [default\_route\_table\_propagating\_vgws](#input\_default\_route\_table\_propagating\_vgws) | List of virtual gateways for propagation | `list(string)` | `[]` | no |
 | <a name="input_default_route_table_routes"></a> [default\_route\_table\_routes](#input\_default\_route\_table\_routes) | Configuration block of routes. See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/default_route_table#route | `list(map(string))` | `[]` | no |
 | <a name="input_default_route_table_tags"></a> [default\_route\_table\_tags](#input\_default\_route\_table\_tags) | Additional tags for the default route table | `map(string)` | `{}` | no |
+| <a name="input_private_route_table_routes_extra"></a> [default\_route\_table\_routes](#input\_default\_route\_table\_routes) | Configuration block of additional routes for private subnets route tables. See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/default_route_table#route | `list(map(string))` | `[]` | no |
+| <a name="input_intra_route_table_routes_extra"></a> [default\_route\_table\_routes](#input\_default\_route\_table\_routes) | Configuration block of additional routes for intra subnets route tables. See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/default_route_table#route | `list(map(string))` | `[]` | no |
```

Comment:
Same here, For accuracy it would be better to use the link to the `aws_route_table` resource instead of `aws_default_route_table`, since this change is affecting resources of the former type. Link is very similar: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route_table#route